tests/attention/test_trtllm_gen_mla.py

```
@@ -1,7 +1,9 @@
+import random
```

Ruff (0.15.10)

[warning] The file is executable but no shebang is present

(EXE002)